test/unit/helpers/MeasureBundleHelpers.test.ts

```diff
@@ -788,10 +789,35 @@ describe('MeasureBundleHelpers tests', () => {
     });
   });
 
-  describe('extractLibrariesFromBundle', () => {
+  describe('extractLibrariesFromLibraryBundle', () => {
+    it('properly gets libraries from EXM130 library bundle', () => {
```

Comment:
I'll defer to @mgramigna on this - is this unit test sufficient, or should we instead try creating a simple bundle fixture that isn't EXM130? I know the unit tests for `extractLibrariesFromMeasureBundle` kinda set the precedent for this, but I think we should try to come up with a simpler example to use so that we start stepping away from using EXM130 for all the tests

Reply:
Yeah good question, we ultimately decided to just re-use the fixture due to the precedent already set in this file. We did discuss an end of sprint activity to scrub the fixtures of these unit tests and make them better, which would involve getting rid of EXM130.

Ultimately, I think since this just re-uses an existing fixture without creating a new one, it's fine, and would be updated along with all the other tests whenever that refactor gets done

Reply:
Cool sounds like a good plan 👍